ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -34,6 +38,8 @@ nginx_sites:
        limit_reqs:
          - zone_name: "server_name"
            nodelay: True
+         - zone_name: "remote_ip_addr"
```

Comment:
This will require a burst attribute since https://github.com/dimagi/commcarehq-ansible/pull/690 requires this now